modules/docs/arrow-docs/docs/docs/effects/async/README.md

```diff
@@ -103,7 +117,7 @@ IO.async().run {
 
 #### defer with CoroutineContext
```

Comment:
A context switch is an effect, that is why you can only do it in the presence of a continuation. For that reason, I believe all `CoroutineContext` references can only appear on `effect` and up but not in `later`.`later` signature will only accept pure functions, pure functions should not fail and a context switch or task submission to a pool can result in a https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/RejectedExecutionException.html . As such `continueOn` should be suspended and can only be accepted as an `effect`

Reply:
The comment above is in regards to APIs like https://github.com/arrow-kt/arrow/pull/1439/files#diff-3d061ce877a7468475fa61a0b9fe6b15R227